includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -168,7 +164,7 @@ public static function should_override_theme() {
 		 *
 		 * @return {bool} The modified learning mode theme override setting.
 		 */
-		return (bool) apply_filters( 'sensei_course_learning_mode_theme_override_enabled', $enabled );
+		return (bool) apply_filters( 'sensei_course_learning_mode_theme_override_enabled', false );
```

Comment:
Do we need to deprecate this filter? From the way I understand it, we want the users' theme styles to always be loaded and never overridden, is that accurate?

Reply:
I think we should deprecate this filter. I wish we could simply remove this filter though.

Reply:
Although I highly doubt that anyone uses this filter. Maybe it's safe to remove it completely? WDYT @yscik ?

If we deprecate the filter instead of removing it, then we should keep everything that uses `should_override_theme` function for at least two more versions.

Reply:
Would be nice to remove all the code, but yeah, it's more considerate to deprecate first. We also heard a few cases where users opted for the theme override, so there is some benefit in keeping it for a while, behind this filter. 
We also still need some of the theme override logic for the site editor anyway.

Reply:
I guess we should leave the filter for now, for the few who may really want to override the theme.

Reply:
Sounds good. Could you add the deprecation tag for it, please. Like `@deprecated $$next-version$$`